stock_location_tank/models/stock_picking.py

```diff
@@ -0,0 +1,165 @@
+# Copyright NuoBiT Solutions - Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    mrp_production_ids = fields.One2many(
+        comodel_name="mrp.production",
+        inverse_name="picking_id",
+        string="Manufacturing Orders",
+    )
+
+    def action_view_mrp_production(self):
+        self.ensure_one()
+        xmlid = "mrp.mrp_production_action"
+        action = self.env["ir.actions.act_window"]._for_xml_id(xmlid)
```

Comment:
It's not necessary to use a variable here, put the XML ID directly to `_for_xml_id` method